website/pages/docs/references/eslint.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: ESLint Plugin
+description: ESLint plugin for Panda CSS 
+---
+
+# ESLint Plugin
+
+ESLint plugin for Panda CSS: `@pandacss/eslint-plugin`.
+
+## Installation
+
+<Tabs items={['pnpm', 'npm', 'yarn', 'bun']} variant="code">
+  <Tab>
+    ```bash
+    pnpm add -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    npm install -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    yarn add -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    bun add -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+</Tabs>
+
+## Rules
```

Comment:
[Suggestion] As someone who is very new to Panda, I don't know what most of these mean by just the one sentence. It'd be nice to have more explanation from them. I'm guessing that's better as a follow-up task. 😄

Reply:
Yeah...I just put something there for now. We definitely have to flesh them up.